GetUpChuckUnityProject/Assets/Scripts/MasterCtrl.cs

```diff
@@ -5,46 +5,228 @@ public class MasterCtrl : MonoBehaviour {
 	//stored reference to red chuck
 	GameObject RedCK;
 	Player RedCKPlayer;
+	Inventory RedInv;
 
 	//stored reference to blue chuck
 	GameObject BlueCK;
 	Player BlueCKPlayer;
+	Inventory BlueInv;
 
 	// Tracks which player is currently active
 	public GameObject activePlayer_go;
 	Player activePlayer;
+	Inventory activeInventory;
 
+	//camera info
+	Camera mainCam;
 	public CameraPan cameraPan;
 
+	//tracks position for mouse drags
+	Vector2 mouseDragStart;
+	Vector2 defMDS = new Vector2 (-1.0f, -1.0f);
+
+	//item pickup ranges
+	public float XEatTol;
+	public float YEatTol;
+
+	//variables for hold for opening inventory in touch
+	bool isIncrementing;
+	int incCount;
+	public int holdTimeForInv;
+
 	// Use this for initialization
 	void Start () {
 		BlueCK = GameObject.FindGameObjectWithTag("blueCK");
 		RedCK = GameObject.FindGameObjectWithTag("redCK");
 		RedCKPlayer = RedCK.GetComponent<Player> ();
 		BlueCKPlayer = BlueCK.GetComponent<Player> ();
-		cameraPan = GameObject.FindGameObjectWithTag("MainCamera").GetComponent<CameraPan>();
+		RedInv = RedCK.GetComponent<Inventory> ();
+		BlueInv = BlueCK.GetComponent<Inventory> ();
+		mainCam = GameObject.FindGameObjectWithTag ("MainCamera").GetComponent<Camera>();
+		cameraPan = GameObject.FindGameObjectWithTag ("MainCamera").GetComponent<CameraPan>();
 		setActivePlayer("red");
 		setActivePlayerGO("red");
+		setInventory ("red");
+		mouseDragStart = defMDS;
+		isIncrementing = false;
+		incCount = 0;
 
 	}
 	
 	// Update is called once per frame
 	void Update () {
-		if (Input.GetButton("CKleft")){
+		/*if (Input.GetButton("CKleft")){
 			walkLeft();
 		}
 		if (Input.GetButton("CKright")){
 			walkRight();
-		}
-		if (Input.GetButtonDown("CKjumpL")){
+		}*/
+		/*if (Input.GetButtonDown("CKjumpL")){
 			jumpLeft();
 		}
 		if (Input.GetButtonDown("CKjumpR")){
 			jumpRight();
-		}
+		}*/
 		if (Input.GetButtonDown("playerSwaps")){
 			swapPlayer();
 		}
+
+#if UNITY_EDITOR
```

Comment:
This should be UNITY_STANDALONE || UNITY_EDITOR, so if we decide to publish for PC/OSX/Linux this region will compile.